### ISSUE TITLE
Sample compression possible?

### ISSUE THREAD

**Issue by Anastasia29** [mreinstein/alexa-utterances#35]
Hi,

I wanted to ask you if there is any way to basically compress the samples. Let's say I have three utterances as input: 
**"why is it happening tonight", 
"where is it happening tonight",
"when is it happening tonight"** 
and as output I want to get an alexa alike utterance syntax that "compresses" my utterances: 
**"(why|where|when) is it happening tonight"**
Is something like this possible to create using your library?  I'd appreciate your feedback and help. Thank you!

Best regards,
Ana

**Comment by mreinstein** [comment]
yes, you are looking for this I believe: https://github.com/alexa-js/alexa-utterances#generate-multiple-versions-of-static-text

**Comment by Anastasia29** [comment]
Hi, If I understand it correctly what you just sent me is kind of other way around. Or not exactly what I need. Could you tell me how can I generate this: **"(why|where|when) is it happening tonight"** from this: 
**"why is it happening tonight",
"where is it happening tonight",
"when is it happening tonight"** ?
What do I need to add to dictionary, slots and my template to achieve this result? I'd very much appreciate your help. 

Thanks and best regards,
Ana

**Comment by mreinstein** [comment]
> how can I generate this: "(why|where|when) is it happening tonight" from this

You can't, it's not currently designed to do that.
